src/Razor/src/Microsoft.VisualStudio.Editor.Razor/TextBufferExtensions.cs

```diff
@@ -3,17 +3,22 @@
 
 #nullable disable
 
-using System;
 using Microsoft.CodeAnalysis.Razor;
 using Microsoft.VisualStudio.Editor.Razor;
 
 namespace Microsoft.VisualStudio.Text
 {
     internal static class TextBufferExtensions
     {
-        public static bool IsRazorBuffer(this ITextBuffer textBuffer!!)
```

Comment:
Felt that we should label this "Legacy" to not give the wrong expectation. Still left the "LSP" portion of the new method added below though just to be extra clear. When we remove the legacy editor entirely we'll be able to remove the legacy API entirely and go from RazorLSP -> Razor

Reply:
Done